docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,268 @@ have not been included. These are:
+  * - Unit Number for any Address (Home OR Mailing OR Employer) 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Miswriting
+    - Noise for all types of addresses will work in the same way 
+  * - PO Box for Mailing Address 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Miswriting
+    - 
+  * - City Name for any Address (Home OR Mailing OR Employer) 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - OCR, phonetic, typographic
+    - Noise for all types of addresses will work in the same way 
+  * - State for any Address (Home OR Mailing OR Employer) 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Incorrect Select
+    - Noise for all types of addresses will work in the same way 
+  * - Zip Code 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 5%
+    - Miswriting
+    - Applies to home, mailing, and employer addresses 
+  * - Relationship to head of household 
+    - Census 
+    - 5%
+    - Incorrect select 
+    - 
+  * - Sex 
+    - Census 
+    - 5%
+    - Incorrect select 
+    - 
+  * - Race/Ethnicity 
+    - Census, WIC
+    - 5%
+    - Incorrect select 
+    - 
+  * - SSN
+    - Taxes (both), SSA
+    - 5%
+    - Miswriting, "borrowed" SSN, Copy from within Household
+    - Note that not all types of noise apply to all observers, details below 
+  * - ITIN
+    - Taxes 1040
+    - 5%
+    - Miswriting, Copy from within Household
+    - Note that not all types of noise apply to all observers 
+  * - Income / Wages
+    - Taxes (both)
+    - 5%
+    - Miswriting
+    - Note that wages and income are on separate tax forms and noise is applied to each separately 
+  * - Employer ID 
+    - Taxes (both)
+    - 5%
+    - Miswriting
+    - 
+  * - Employer Name 
+    - Taxes (both)
+    - 5%
+    - OCR, typographic 
+    - 
+  * - Type of Tax Form  
+    - Taxes (both)
+    - 5%
+    - Incorrect select 
+    - 
+  * - Type of SSA Event 
+    - SSA 
+    - 5%
+    - Incorrect select 
+    - 
+  * - Date of SSA Event 
+    - SSA 
+    - 5%
+    - Miswriting, month and day swap 
+    - 
+
+The below section further describes types of noise including any code 
+available and information for implimentation. **Software engineering team - please alert the research team if any of the below looks to be particularly challenging for further discussion.**
+
+**OCR**
+
+Optical character recognition is when a character is misread for another character that 
+looks similar. A common examples is 'S' and '5'. In order to emulate 
+that, there is a `GeCo like corrupter and related list of possible changes in the ocr_variations_upper_lower csv found here <https://github.com/ihmeuw/vivarium_research_prl/tree/main/noise>`_. 
+
+Currently, the user defined rate will be a rate 'eligible' for noise. The actual rate of noise 
+will be determined at the character level as XX errors per character. We think this approximates to 1 
```

Comment:
What is XX here?

Reply:
oh, I see the todo about it below.

Reply:
Yep! I think Nathaniel and I still need to work out what value this is going to be so its a placeholder for now.

Reply:
Do we want approximately 1 error per selected string? That seems like it might be a good reasonable default, but I hadn't really thought about it. On the other hand, @zmbc [suggested](https://github.com/ihmeuw/vivarium_research/pull/1088/files#r1047510209) that maybe it's better to have the user specify the per-character error probability.

If we *do* want approximately 1 error per selected string, we could have a column-dependent value of XX, determined something like this (some of Zeb's comments gave me this idea):

1. Compute the average length of strings in the column (or in a random sample of strings in the column that will be eligible for error)
2. Choose XX to be 1/(average string length)
3. Choose the fraction of eligible rows to be
```
(desired fraction of rows with error) / (probability of at least one error),
```
which, as Zeb pointed out [below](https://github.com/ihmeuw/vivarium_research/pull/1088#discussion_r1047504927), is approximately
```
(desired fraction of rows with error) / 0.65
```
if the average string length is at least 5. The denominator would be larger for shorter strings (1 for length 1, 0.75 for length 2,...) and approaches 1-1/e = 0.632... as the average string length approaches infinity.

In this case the average number of corrupted characters in a string ,given that there is at least one error, would be approximately 1/0.65 = 1.53.... Note that there's no way to make this value exactly equal to 1 since the minimum number of errors is 1. (However, we could drive the average arbitrarily close to 1 by artificially inflating the number of strings with exactly 1 error. That is, we could modify the individual noise functions to make errors in exactly one character more likely than would occur when all characters independently have the same probability of corruption. But that seems too complicated for a first pass.)

With this strategy, the user could specify the actual fraction of rows they want errors in, instead of just the eligible fraction.

Except that this strategy won't quite work for OCR and phonetic noise, which have a "per-token" error probability instead of a per-character probability (tokens are up to 3 characters for OCR and up to 7 characters for phonetic).

Reply:
So yeah, it's actually not accurate to say that we have a "rate of XX errors per character," because it's really a rate per *token*, and tokens have variable length. Sorry, @SylLutze, for not being more precise when I described how the noise functions work!

I think the variable-length token problem was one of the main reasons I was saying we should leave the probabilities for the noise functions as an implementation detail to figure out later.